modules/index-service/src/test/java/org/opencastproject/index/service/impl/IndexServiceImplTest.java

```diff
@@ -754,25 +738,12 @@ public void testCreateEventInputNormalExpectsCreatedRecurringEvent() throws Exce
     assertTrue("The mediapackage should have had its title updated", catalogResult.hasCaptured());
     assertEquals("The mediapackage title should have been updated.", expectedTitle, mediapackageTitleResult.getValue());
     assertTrue("The catalog should have been created", catalogResult.hasCaptured());
-    //Assert that the start and end recurrence dates captured, along with the duration and recurrence rule
-    //This is all used by the scheduling calculation, but not the actual scheduling call
-    assertTrue(recurrenceStart.hasCaptured());
-    assertTrue(recurrenceEnd.hasCaptured());
-    assertTrue(duration.hasCaptured());
-    assertTrue(rrule.hasCaptured());
     //Assert that the scheduling call has its necessary data
     assertTrue(schedStart.hasCaptured());
     assertTrue(schedEnd.hasCaptured());
     assertTrue(schedDuration.hasCaptured());
     assertTrue(schedRRule.hasCaptured());
     assertTrue(schedTz.hasCaptured());
-    List<Period> pCheck = calculatePeriods(schedRRule.getValue(), schedStart.getValue(), schedEnd.getValue(), (Long) schedDuration.getValue(), schedTz.getValue());
-    List<Period> pExpected = calculatePeriods(rrule.getValue(), recurrenceStart.getValue(), recurrenceEnd.getValue(), (Long) duration.getValue(), tz.getValue());
-
-    //Assert that the first capture time is the same as the recurrence start
-    assertEquals(pExpected.get(0).getStart(), pCheck.get(0).getStart());
-    //Assert that the end of the last capture time is the same as the recurrence end
-    assertEquals(pExpected.get(pExpected.size() - 1).getEnd(), pCheck.get(pCheck.size() - 1).getEnd());
   }
```

Comment:
Parts of this test case were based on the unnecessary call of `calculatePeriods()` in schedulerServiceImpl